lib/cubit/todolist_cubit.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:bloc/bloc.dart';
+
+import '../models/todo.dart';
+
+class TodolistCubit extends Cubit<List<Todo>> {
```

Comment:
even though it's true that it's just a list I think it's better to define the state as a separate class like `TodoListCubitState` to hold the state and it will make it easy to add new things in case if needed